requirements/edx/base.txt

```diff
@@ -136,6 +136,7 @@ freezegun==0.1.11
 lettuce==0.2.20
 mock-django==0.6.6
 mock==1.0.1
+moto==0.3.1
```

Comment:
I think that would greatly increase the scope of this story quite a bit, since we're using `boto` in several different parts of the codebase.  Would it be okay to create a separate JIRA ticket to track the work to upgrade both boto and moto?